gui/control_main.py

```diff
@@ -4852,7 +4852,7 @@ def initUI(self):
         self.immediate_comm_pv = PV(daq_utils.beamlineComm + "immediate_command_s")
         self.stillModeStatePV = PV(daq_utils.pvLookupDict["stillModeStatus"])
         self.progressDialog = QtWidgets.QProgressDialog()
-        self.progressDialog.setCancelButtonText(QString("Cancel"))
+        self.progressDialog.setCancelButtonText("Cancel")
```

Comment:
according to the documentation, it seems to be ok to use regular text. was there any reason why QString was used before?

Reply:
I have seen this in at least one other place, perhaps requirements changed in Qt or PyQt some time ago...

Reply:
you provided the explanation that this might have been an old requirement for the buttons and what it was being covered up with "QString = str"... resolving.